Gems/Multiplayer/Code/Source/Pipeline/NetworkPrefabProcessor.cpp

```diff
@@ -183,6 +186,11 @@ namespace Multiplayer
             context,
             networkSpawnableAssetId);
 
+        if (networkInstance.GetEntityAliasCount() == 0 && networkInstance.GetNestedEntityAliasCount() == 0)
```

Comment:
You can just use GetNestedEntityIds() and provide a callback that increments count for this purpose.

Reply:
I just noticed that GetEntityAliasCount exists, so seems like we should do the same for GetNestedEntityAliasCount. Iterating each would work, but felt too obfuscated. Thoughts?

Reply:
What you are getting here is the count of number of nested instances, which won't give you the correct number of nested entities anyway. Each nested instance can have any number of entities under them. You'd need to implement a recursive logic like GetNestedEntitiyIds() does. But since that function already exists and serves the exact problem you are looking to use, you can avoid creating this new function altogether.

Reply:
Thank you! That explanation helps :)
Checkout the latest, I just track network entities in the pre-existing loop. Added a unit test as well called NetworkPrefabProcessor_ProcessPrefabEntityHierarchy_NestedNetworkEntityIsFound